config/after_symlink_shared.yml

```diff
@@ -1,17 +1,41 @@
-- import_tasks: "../../cbrunnkvist.ansistrano-symfony-deploy/config/steps/composer.yml"
+- include_tasks: "{{ ansistrano_symfony_before_composer_tasks_file | default('steps/empty.yml') }}"
+
+- include_tasks: "../../cbrunnkvist.ansistrano-symfony-deploy/config/steps/composer.yml"
```

Comment:
Almost :) Please revert the changes to the local `/steps/*.yml` files to `import_tasks` since those are static imports.